photutils/datasets/sources.py

```diff
@@ -37,7 +39,7 @@ def make_model_params(shape, n_sources, flux_range, *, min_separation=1,
         given ``shape`` and therefore the number of sources generated
         may be less than ``n_sources``.
 
-    flux_range : 2-tuple
+    flux_range : 2-tuple, optional
```

Comment:
actually it is not optional. It has a default value :-) optional means that the default value is None

Reply:
@VascoSch92 "Optional" means that the user does not need to specify a value (e.g., all keywords are "optional" in numpydoc style).  Please see the `numpydoc` style doc:  https://numpydoc.readthedocs.io/en/latest/format.html